src/morkparser.cpp

```diff
@@ -802,5 +802,5 @@ int MorkParser::dumpMorkFile( const QString& filename )
         printf("\n\n" );
     }
 
-    return 0;
+    return EXIT_SUCCESS;
```

Comment:
I forgot to commit this in #162 so I'm including it here. `EXIT_SUCCESS` is defined to `0`, so this is **not** a change in behaviour.